contracts/factories/PairFactory.sol

```diff
@@ -30,8 +36,38 @@ contract PairFactory is IPairFactory {
         pauser = msg.sender;
         isPaused = false;
         feeManager = msg.sender;
-        stableFee = 2; // 0.02%
-        volatileFee = 2;
+        stableFee = 3; // 0.03%
+        volatileFee = 25; // 0.25%
+        deployer = msg.sender;
+    }
+
+    function setTeam(address _team) external {
+        require(msg.sender == deployer); // might need to set this to deployer?? or just make it
+        require(team == address(0), 'The team has already been set.');
+        team = _team;
+    }
+
+    function setVoter(address _voter) external {
+        require(!initial_voter_set, 'The voter has already been set.');
```

Comment:
@0xMotto we don't really need initial_voter_set and initial_tank_set? we can just check the address is 0?

Reply:
its good if we can maintain setTank (require team) so we can adjust it later

Reply:
removed `initial_voter_set ` and `initial_tank_set `.